pkg/limayaml/validate.go

```diff
@@ -489,13 +496,17 @@ func validateParamIsUsed(y *limatype.LimaYAML) error {
 		}
 		keyIsUsed := false
 		for _, p := range y.Provision {
-			if re.MatchString(p.Script) {
-				keyIsUsed = true
-				break
-			}
-			if p.Content != nil && re.MatchString(*p.Content) {
-				keyIsUsed = true
-				break
+			for _, ptr := range []*string{
+				&p.Script,
+				// mode: data
+				p.Content, p.Owner, p.Path,
+				// mode: yq
+				p.Expression, p.Permissions,
```

Comment:
`p.Permissions` is also part of "mode: data". Maybe it is easiest to remove the comments and have just a list?